Packs/ReversingLabs_Titanium_Cloud/Integrations/ReversingLabsTitaniumCloudv2/ReversingLabsTitaniumCloudv2.yml

```
@@ -311,9 +311,27 @@ script:
       default: true
       required: true
     outputs:
+    - contextPath: File.MD5
+      description: MD5 hash.
+      type: Unknown
```

Why `Unknown` here? Shouldn't it be:
```suggestion
      type: String
```

Same for some of the other fields (`File.SHA1`, `File.SHA256`)

---

Thank you! I am aware of the failing checks and I am currently resolving them.